package.json

```diff
@@ -20,7 +20,8 @@
     "clients:rust:lint": "zx ./scripts/client/lint-rust.mjs",
     "clients:rust:publish": "zx ./scripts/client/publish-rust.mjs",
     "clients:rust:test": "zx ./scripts/client/test-rust.mjs",
-    "template:upgrade": "zx ./scripts/upgrade-template.mjs"
+    "template:upgrade": "zx ./scripts/upgrade-template.mjs",
+    "spellcheck:rust": "cargo spellcheck --code 1"
```

Comment:
Should we call this `rust:spellcheck` in preparation for febo's work on unifying scripts?

@febo After your refactoring, I think this should be part of the `lint` subtasks, so it would probably need to be located in `scripts/rust/lint-spellcheck`. Lmk what you think.

Reply:
Sounds good!